schema/home/index.js

```diff
@@ -8,8 +10,10 @@ import {
 const HomePageType = new GraphQLObjectType({
   name: 'HomePage',
   fields: {
-    modules: HomePageModules,
```

Comment:
Aye, that was my thinking too.

Eventually you can move over to these ones and we can remove the other root fields.